baseline-error-prone/build.gradle

```
@@ -7,6 +7,7 @@ apply from: "${rootDir}/gradle/publish-jar.gradle"
 
 dependencies {
     compile 'com.google.errorprone:error_prone_core'
+    compile 'org.mockito:mockito-errorprone'
```

Might be cleaner to provide it as a runtime dependency since nothing compiles against it. baseline-error-prone isn't used as a library dependency, so it doesn't matter much either way
```suggestion
    runtimeOnly 'org.mockito:mockito-errorprone'
```